blazor/datagrid/cell.md

```diff
@@ -134,69 +210,79 @@ The following screenshot represents a DataGrid with customized cell styles.
-        {
-            RollNo = 1000 + x,
-            Name = (new string[] { "ALFKI", "ANANTR", "ANTON", "BLONP", "BOLID" })[new Random().Next(5)],
-            PatentFamilies = 1000 + x * 5,
-            Country = (new string[] { "Australia", "Japan", "Canada", "India", "USA" })[new Random().Next(5)],
-            MainFields = (new string[] { "Printing, Digital paper, Internet, Electronics,Lab-on-a-chip, MEMS, Mechanical, VLSI", "Various", "Printing, Digital paper, Internet, Electronics, CGI, VLSI", "Automotive, Stainless steel products", "Gaming machines" })[new Random().Next(5)],
-        }).ToList();
+            {
+                OrderID = 1000 + x,
+                CustomerID = (new string[] { "ALFKI", "ANANTR", "ANTON", "BLONP", "BOLID" })[new Random().Next(5)],
+                Freight = 2.1 * x,
+                OrderDate = DateTime.Now.AddDays(-x),
+            }).ToList();
     }
 
     public class Order
     {
-        public int? RollNo { get; set; }
-        public string Name { get; set; }
-        public int? PatentFamilies { get; set; }
-        public string Country { get; set; }
-        public string MainFields { get; set; }
+        public int? OrderID { get; set; }
+        public string CustomerID { get; set; }
+        public DateTime? OrderDate { get; set; }
+        public double? Freight { get; set; }
     }
 }
 ```
 
-The following screenshot represents a DataGrid with auto wrap.
+{% previewsample "https://blazorplayground.syncfusion.com/embed/LNLANFtEpCOkMFqY?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
 
-{% previewsample "https://blazorplayground.syncfusion.com/embed/BNrgtxrihfvRzHnK?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
+### Using property
 
-## Custom attributes
+To customize the style of grid cells, define [CustomAttributes](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.ColumnModel.html#Syncfusion_Blazor_Grids_ColumnModel_CustomAttributes)  property to the GridColumn  definition object. The [CustomAttributes](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.ColumnModel.html#Syncfusion_Blazor_Grids_ColumnModel_CustomAttributes)  property takes an object with the name-value pair to customize the CSS properties for grid cells. You can also set multiple CSS properties to the custom class using the customAttributes property.
 
-You can customize the DataGrid cells by adding a CSS class to the [CustomAttributes](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.ColumnModel.html#Syncfusion_Blazor_Grids_ColumnModel_CustomAttributes) property of the column.
-
-In the following example, the cells of the **OrderID** and **ShipCity** columns are customized.
+```cshtml
+<style>
+    .e-attr {
+        background: #d7f0f4;
+    }
+</style>
+```
+The following example demonstrates how to customize the appearance of the **OrderID** and **ShipCity** columns using custom attributes.
```

Comment:
Some code and paragraph missing from angular UG

Reply:
added the missing line